pages/android-pc.md

```diff
@@ -2,95 +2,42 @@
-the magnetic declination should be entered to align the map with magnetic north.
-You may simply use the lookup button to look it up on the internet for the place
-of the template you loaded. However, be aware that the lookup only uses a model
-which may deviate from the real declination. It may be advantageous to check
-this in the terrain. All remaining values in the georeferencing dialog are
-already initialized using the newly loaded template, so you can then finish the
-dialog.
-
-Then load all remaining templates and finish preparing the map.
-Non-georeferenced templates will need to be moved to match the location of the
-georeferenced templates.
-
-
-Georeferencing when you use an existing map
--------------------------------------------
-
-In this case, the easiest way is to first create a new map with the same symbol
-set as the existing map. The old map can be selected as the symbol set source in
-the new map dialog to do this. Then load a georeferenced template as detailed in
-the paragraph above (use e.g. OpenStreetMap if you do not have a georeferenced
-template).
-
-After that, load the existing map as template in the new map. Then use the
-alignment tools for templates to move the existing map to match with the new
-georeferenced template. When you are finished, import the existing map template
-into the new map file using the corresponding action in the template dock widget,
-and you should be done.
-
-Attention: especially for old maps, the whole existing map may be locally
-distorted. In this case it will not be possible to match the old map and the
-georeferenced template and the only way to georeference the map is to do a
-tedious undistortion. Mapper does not provide a tool for this.
-
-
-Preparing for free-hand scribbling
-----------------------------------
-
-![](../mapper-images/pencil.png)
-
-OO Mapper supports free-hand drawing onto image templates using the tool with
-the pencil icon shown above. This is especially convenient to use in case you
-want to do only (or mostly) drafting while you are in the field, and draw the
-final version of the map at home on a PC. It has the advantage of being quick
-and it only alters the template you draw on, i.e. it does not create new map
-objects, so Mapper's map display does not become slower. This may be relevant on
-mobile devices because of little processing power or because of battery runtime.
-
-If you want to use this functionality, you should best load a separate,
-initially transparent image template in png format for it. Suited images can be
-downloaded here:
-
- - <a href="attachment/scribble_1024.png">size 1024x1024</a>
- - <a href="attachment/scribble_2048.png">size 2048x2048</a>
+The Android app cannot be used to create new maps because it would be cumbersome to setup the base maps there. Instead, maps should be prepared with templates and possibly georeferencing using the PC version of Mapper first and then transferred to the mobile device for surveying.
```

Comment:
Do we really dislike the fixed line breaks that much?

Reply:
Feels like something that should be left for peoples text editor settings, when built with Karmadown it just ends up being weird HTML. Note that spaces was missing next to each line break(although the browser renders them as such).